lib/makeTwemojiRenderer.js

```diff
@@ -60,4 +53,25 @@ function merge() {
 		}
    }
    return out
+}
+
+function getSource(options, icon){
+	var source = ''
+
+	if (options.baseUrl.indexOf('http') !== 0) {
+		source += normalizeProtocol(options.protocol)
+	}
+
+	source += options.baseUrl
+
+	if(options.size === '' && options.baseUrl.endsWith('/') && options.ext === '.svg'){
```

Comment:
I think adding svg to the url should be more explicit, and not an implicit behavior from passing `size: ''`. So I think a more robust solution would require a larger API change. We're also very sensitive to the bundle size so I'd suggest a simpler fix of simply dropping the double slash with minimal modifications to line 36. This way our existing API would work more as expected without any surprises.

Another consideration is: would making this change break the issue reporter's existing bug fix https://github.com/line/armeria/pull/4646/files#diff-2f901e80d43fc4fbadc238e158e1f4f161fbc9baf23d030dfa4e331a1b0e3b0c